src/deluge/gui/menu_item/defaults/pad_brightness.h

```diff
@@ -23,12 +23,21 @@ namespace deluge::gui::menu_item::defaults {
 class PadBrightness final : public Integer {
 public:
 	using Integer::Integer;
-	[[nodiscard]] int32_t getMinValue() const override { return kMinLedBrightness; }
-	[[nodiscard]] int32_t getMaxValue() const override { return kMaxLedBrightness; }
-	void readCurrentValue() override { this->setValue(FlashStorage::defaultPadBrightness); }
+	void selectEncoderAction(int32_t offset) override {
+		offset = offset * 4;
```

Comment:
Offset is already multiplied by 5 if you're holding shift - this shouldn't be needed

Reply:
OK maybe I can add this in the doc as testers were complaining about it and removing the *4 offset

Reply:
this *4 is serving a different purpose, preventing the selection of brightness values that don't map to a valid interval setting. I think it's ok as is.

Reply:
Right, ok that makes sense